src/app/admin/mod.rs

```diff
@@ -3,32 +3,38 @@
 //! * `/metrics` -- reports prometheus-formatted metrics.
 //! * `/ready` -- returns 200 when the proxy is ready to participate in meshed traffic.
```

Comment:
I think the /proxy-log-level route should be included here

Reply:
whoops, good eye!